tests/integration/test_gitlab.py

```diff
@@ -34,6 +34,15 @@ def setUp(self):
 
 
 class GenericCommands(GitlabTests):
+    def test_get_file_content(self):
+        project = self.service.get_project(repo="hello-world", namespace="shreyaspapi")
+
+        file = project.get_file_content(
+            path="test.md", ref="825abfaafb786aa9abbf7093cbe21dbc8d018277"
+        )
```

Comment:
Can we reuse some project we already use (e.g `self.project` -> `packit-service/ogr-tests`)?